packages/animations/lib/src/dual_transition_builder.dart

```diff
@@ -16,7 +16,7 @@ import 'package:flutter/widgets.dart';
 typedef TransitionBuilder = Widget Function(
   BuildContext context,
   Animation<double> animation,
-  Widget child,
+  Widget? child,
```

Comment:
To avoid that this one and the one in the framework diverge: Can you just remove this file all together and use the DualTransitionBuilder from flutter/flutter (see TODO at top of this file)?

Reply:
Done.